docs/visualize/vega.asciidoc

```diff
@@ -16,21 +16,20 @@ NOTE: In Vega it is possible to load data dynamically, e.g. by setting signals a
 [[vega-quick-demo]]
 === Getting Started with Vega
 
-Follow these steps to create your first Vega visualization.
-
-. In Kibana, choose Visualize, and add Vega visualization. You should see a default graph.
-. Try changing `mark` from `line` to `point`, `area`, `bar`, `circle`,
-`square`, ... (see
-https://vega.github.io/vega-lite/docs/mark.html#mark-def[docs])
-. Try other https://vega.github.io/vega/examples/[Vega] or
-https://vega.github.io/vega-lite/examples/[VegaLite] visualizations. You
-may need to make URLs absolute, e.g. replace
+* Go to *Visualize*, click *+*, and select *Vega* to see an example graph.
+* Try changing `mark` from `line` to `point`, `area`, `bar`, `circle`,
+or `square`. Check out the 
+https://vega.github.io/vega-lite/docs/mark.html#mark-def[Vega docs] for more information.
+* Explore other available https://vega.github.io/vega/examples/[Vega] or
+https://vega.github.io/vega-lite/examples/[VegaLite] visualizations. 
```

Comment:
```suggestion
https://vega.github.io/vega-lite/examples/[Vega-Lite] visualizations. 
```

Reply:
Great, thanks - I found a few more instances of VegaLite throughout the doc, so I updated all of them.